### ISSUE TITLE
Add Bug Report / Feature Request mechanism

### ISSUE THREAD

**Issue by elias-pap** [UoC-Radio/flow-dashboard#3]
This can be a main menu "Help..." entry, which leads to a window pop-up. In there the user can type the message, which will be sent by e-mail to eliaspap@pm.me

**Comment by ggalan87** [comment]
This requires interaction with e-mail server, which will be unnecessary dependency. I think it is a better idea to just open the default mail client desktop app, as it is usually done.

**Comment by elias-pap** [comment]
True. An even better idea might be something like [mailgun](https://www.mailgun.com/).
